docs/typescript/production-deploy.md

```diff
@@ -131,11 +131,91 @@ We endeavor to give you good defaults, so you don't have to worry about them, bu
   - Note that there are separate [Timeouts and Retry Policy](https://typescript.temporal.io/api/interfaces/client.workflowoptions/#workflowruntimeout) at the Workflow level, but we do not encourage their usage unless you know what you are doing.
 - _to be completed as we get more user feedback_
 
-## Do not use Alpine
+## Running in Docker
+
+Workers based on TypeScript SDK can be deployed and run as Docker containers.
+
+At this moment, we recommend usage of NodeJS 16 (note that there are known issues with NodeJS 18). Both `amd64` and `arm64` platforms are supported. A glibc-based image is required; musl-based images are _not_ supported (see below).
+
+The easiest way to deploy a TypeScript SDK Worker on Docker is to start with the `node:16-bulleyes` image. For example:
+
+```dockerfile
+FROM node:16-bulleyes
+
+COPY . /app
+WORKDIR /app
+
+RUN npm install --only=production \
+    && npm run build
+
+CMD ["build/worker.js"]
+```
+
+For smaller images and/or more secure deployments, it is also possible to use `node:slim` Docker image variants (eg. `node:16-bulleyes-slim`), or `distroless/nodejs` Docker images (ie. `grc.io/distroless/nodejs:16`). See details below.
+
+### Using `node:slim` images
+
+`node:slim` images do not contain some of the common packages found in regular images. This results in significantly smaller images.
+
+Note however that TypeScript SDK requires the presence of root TLS certificates (ie. the `ca-certificates` package), which are not included in `slim` images. This package is required even when connecting to a local Temporal Server, and when using a server connection config that doesn't explicitly use TLS.
+
+For this reason, the `ca-certificates` package must be installed during the construction of the Docker image. For example:
+
+```dockerfile
+FROM node:16-bulleyes-slim
+
+RUN apt-get update \
+    && apt-get install -y ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+# ... same as with regular image
+```
+
+Failure to install this dependency results in `[TransportError: transport error]` runtime error, because the certificates cannot be verified.
```

Comment:
```suggestion
Failure to install this dependency results in a `[TransportError: transport error]` runtime error, because the certificates cannot be verified.
```